spec/models/game_spec.rb

```diff
@@ -117,5 +117,21 @@
       end
     end 
   end
+
+  describe 'game end' do
+    it 'should set finished to 1 on game end' do
```

Comment:
Should "1" be "true"? (I'm guessing you rewrote the test but forgot to adjust the name!)

Reply:
Good catch Jared. Looking into it right now

Reply:
Good catch Jared. Looking into this right now